namespaces/live-1.cloud-platform.service.justice.gov.uk/sentence-planning-preprod/03-resourcequota.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: ResourceQuota
+metadata:
+  name: namespace-quota
+  namespace: sentence-planning-preprod
+spec:
+  hard:
+    requests.cpu: 3000m
```

Comment:
Please reduce requests.cpu for the namespace to 100m

This is not a limit, it's just the amount of CPU the cluster will reserve for your namespace, on creation.

Reply:
changed this for dev and preprod